### PR TITLE
removed uneccessary statements and auto-formatted

### PR DESCRIPTION
**Pull request by nimendrak**

### CODE REVIEW

**Comment by nimendrak** [comment]
I could not remove the progress bar from the console but every other thing was sorted out.

**Comment by lucasace** [comment]
Dont worry the progress bar is fine, its because of tqdm

**Comment by nimendrak** [comment]
Thanks!

File: Final Project/check.py [hunk 1]
```diff
@@ -1,12 +1,16 @@
+import fastai as fastai
```
Comment (lucasace) [inline Final Project/check.py:1]: may i know what these two header files are

Comment (lucasace) [reply Final Project/check.py:1]: the other one being spark

**Review by lucasace — COMMENTED** [review]
We didnt have any hidden folder called idea, can u elaborate on this, is this because of the autoformatter?
if yes then reformate it using black only

File: Final Project/team.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 
 def teams(Id, value=5):
-    def predict_score(player, position, date):  # position: batsman, all-rounder, wk, bowler
+    def predict_score(player, position, date):
         # date format yyyy-mm-dd
         # When assigning position, make sure the file exists in github repo
```
Comment (lucasace) [inline Final Project/team.py:12]: Delete this comment line

**Review by lucasace — CHANGES_REQUESTED** [review]

File: Final Project/team.py (hunk 2 above)
Comment (lucasace) [reply Final Project/team.py:12]: the comment line," #when assigning position" has to be deleted

File: Final Project/check.py (hunk 1 above)
Comment (nimendrak) [reply Final Project/check.py:1]: This is the library that I tried to use for hide progress bar in console, You can remove it.

File: Final Project/team.py (hunk 2 above)
Comment (nimendrak) [reply Final Project/team.py:12]: Alright!

File: Final Project/check.py (hunk 1 above)
Comment (lucasace) [reply Final Project/check.py:1]: Could you remove both the libraries as it is not required

**Comment by nimendrak** [comment]
> We didnt have any hidden folder called idea, can u elaborate on this, is this because of the autoformatter?
> if yes then reformate it using black only

It is a PyCharm's auto generated folder that keeps the metadata of the project. If you using another IDE, You can remove it as well.

**Comment by lucasace** [comment]
Aah okay, once the other changes are done , delete this folder as well, as  it is not neccessary

**Comment by nimendrak** [comment]
> Aah okay, once the other changes are done , delete this folder as well, as it is not neccessary

Sure!

File: Final Project/check.py (hunk 1 above)
Comment (nimendrak) [reply Final Project/check.py:1]: Done!

Comment (lucasace) [reply Final Project/check.py:1]: I dont think you have commited it to your repo, i cannot see any change here

**Review by lucasace — APPROVED** [review]

**Comment by lucasace** [comment]
> > Aah okay, once the other changes are done , delete this folder as well, as it is not neccessary
> 
> Sure!

Complete this and you are good to go

**Comment by nimendrak** [comment]
Thanks @lucasace !

**Review by lucasace — CHANGES_REQUESTED** [review]
Heyy a small thing,I could use a confirmation commit that states auto-formatting using black 
Black is a package which automatically formats your code, the link to its docs is given in the issue
This is only because, I asked you to make some changes, after which auto-formatting was not done

**Comment by nimendrak** [comment]
I've added the Black package but Let me check it again.

**Review by lucasace — APPROVED** [review]

**Comment by lucasace** [comment]
Great job @nimendrak , Thanks for contributing!

**Pull request merged**